src/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CTAButton from '../CTAButton/CTAButton';
+import {
+  HeroSectionWrapper,
+  HeroSectionTitle,
+  HeroSectionP,
+  HeroSectionButtonLinkWrapper,
+  Table,
+  TableContainer,
+} from './HeroSection.styles';
+
+const HeroSection = () => (
+  <HeroSectionWrapper>
+    <HeroSectionTitle>No reactions to your reddit posts?</HeroSectionTitle>
+    <HeroSectionP>
+      Great timing, great results! Find the best time to post on your subreddit.
+    </HeroSectionP>
+    <HeroSectionButtonLinkWrapper to="/search?q=javascript">
+      <CTAButton type="button" cta="SHOW ME THE BEST TIME" />
```

Comment:
Do you need `type="button"` here?

Reply:
I assumed that it's a good practice to set type for buttons but after some googling I don't think it's obligatory to specify it

Reply:
I wasn't sure as well. Was asking out of curiosity ;-)